ess/src/API/EMBC.ESS/Resources/Contacts/Contract.cs

```diff
@@ -37,6 +37,7 @@ public class ContactQuery
     {
         public string ByContactId { get; set; }
         public string UserId { get; set; }
+        public bool LeaveSecurityAnswersUnmasked { get; set; } = false;
```

Comment:
How about 'MaskSecurityAnswers' default to 'true'?

Reply:
@ytqsl updated